src/upstart/ceph-osd.conf.in

```diff
@@ -15,7 +15,7 @@ pre-start script
 
     install -d -m0770 -o ceph -g ceph /var/run/ceph
 
-    /usr/lib/ceph/ceph-osd-prestart.sh --cluster="${cluster:-ceph}" -i "$id"
+    @CMAKE_INSTALL_FULL_LIBEXECDIR@/ceph/ceph-osd-prestart.sh --cluster="${cluster:-ceph}" -i "$id"
```

Comment:
as mentioned about, i'd suggest we stop improving upstart script in master branch.